### DISCUSSION TITLE
Dependency track - login issue

### DISCUSSION THREAD

**Discussion by rubi122** [DependencyTrack/dependency-track#889] [General]
Hi,

I'm trying to complete the installation of dependncytrack.
I'm not export in Linux, but I have installed the docker on Ubuntu machine, and I can rich the logon screen of dependcnytracker, but when I'm entering the user and password I'm getting an error message
"Failed to determine the availability of OpenID Connect."
Any idea how should I continue from [here?](https://ibb.co/SwZJBkq)

**Comment by stevespringett** [comment]
When accessing Dependency-Track remotely, you'll need to specify `API_BASE_URL` for the frontend image so that it knows how to reach the backend. The simplest way is to modify the value of`API_BASE_URL` in docker-compose.yml.

See also: https://github.com/DependencyTrack/dependency-track/issues/887
